src/test/java/seedu/address/logic/commands/GetMonthlyFeeCommandTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.TypicalStudents.ALICE;
+import static seedu.address.testutil.TypicalStudents.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.fee.Month;
+import seedu.address.model.fee.Year;
+
+public class GetMonthlyFeeCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    // Alice has 4 session and has only 2 session that is on 1 Jan 2021
+    // The other 2 session, one has a different month, one has a different year
+    @Test
+    public void execute_correctMonthlyFee() throws CommandException {
+        GetMonthlyFeeCommand getMonthlyFeeCommand = new GetMonthlyFeeCommand(ALICE.getName(),
+            new Month(2), new Year(2021));
+        CommandResult commandResult = getMonthlyFeeCommand.execute(model);
+
+        assertEquals("Monthly fee for Alice Pauline on February, 2021 is $120.90",
```

Comment:
I'm not sure if we should hardcode this, maybe run some codes to calculate Alice's fees for feb 2021 then concatenate it with the expectedResult?

I'm just afraid someone amends the TypicalStudent's sessions and this will screw up. What do you think?

Reply:
Hmmm ok sure! I have made the amendments, feels like I am doing a calculation outside, and then a calculation inside the command and comparing them. But ok ah, don't think it will ever fail now.